README.md

```diff
@@ -1,69 +1,19 @@
 # Yet Another Gamma Index Tool
 
-Library written in C++ for performing efficient comparisons of DICOM images using 2D, 2.5D and 3D gamma index concept.
+YAGIT is a library written in C++ for performing efficient comparisons of two images
+containing radiation dose distributions using the 2D, 2.5D, and 3D gamma index concepts.
```

Comment:
```suggestion
YAGIT is a C++ library for performing efficient comparisons of two images
containing spatial dose distributions. The library employs 2D, 2.5D, and 3D gamma index concepts.
```

At the end of last sentence I'd add a link to some scientific paper(s?) where gamma index is defined. The users of this library are scientists. They should be accustomed to such type of source materials